index.html

```diff
@@ -53,6 +53,149 @@ <h1  class="headline1">I'm Loukmane
+                <p>css</p>
+                <p>javaScript</p>
+              </div>
+              <div class="view-project">
+              <p>See Project</p>
+            </div>
+              </li>
+              <li class="work work4">
+                <img class="img1" src="portfolioimgs/Snap4.png" alt="work1Snapshot">
+                  <h1 class="multi">Multi-Post
+                    Stories</h1>
+                  <div class="canopy">
+                  <h3>CANOPY</h3>
+                  <ul class="cards">
+                    <li class="info">Back End Dev</li>
+                    <li class="info">2015</li></ul>
+                  </div>
+                  <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                 <div class="tools"> <p>html</p>
+                  <p>css</p>
+                  <p>javaScript</p>
+                </div>
+                <div class="view-project">
+                <p>See Project</p>
+              </div>
+                </li>
+         </ul>
+       </section>
+
+       <section class="about-section">
+        <div class="about">
+         <h1 class="aboutMySelf" >About<br>Myself</h1>
+          <p class="text text2 ">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+<p class="contact">LET'S CONNECT</p>
+<ul class="box">
+<li class="socialMedia">
+    <a href="https://twitter.com/LoukmaneOustani">
+      <img src="portfolioimgs/1.png" alt="twitter">
+     </a>
+ </li>
+ <li class="socialMedia">
+   <a href="https://www.linkedin.com/in/loukmane-oustani-221668211/">
+     <img src="portfolioimgs/2.png" alt="Linkedin">
+   </a>
+ </li>
+ <li class="socialMedia">
+   <a href="#">
+     <img src="portfolioimgs/3.png" alt="M">
+   </a>
+ </li>
+ <li class="socialMedia">
+   <a href="https://github.com/L2oukmane">
+     <img src="portfolioimgs/4.png" alt="github">
+   </a>
+ </li>
+ <li class="socialMedia">
+   <a href="#">
+     <img src="portfolioimgs/5.png" alt="peace">
+   </a>
+ </li>
+</ul>
+        </div>
+<div class="resume">
+  <p>Get my resume</p>
+  <div class="languages">
+ <h4>Languages</h4>
+ <img class="scrollDown" src="portfolioimgs/down.png" alt="down">
+</div>
+
+  <ul class="html-css">
+    <li class="html-container">
+   <img  class ="html" src="portfolioimgs/HTML.png" alt="html"/> <span class="html-li">HTML</span>
+    </li>
+  <li class="css-container">
+   <img class="css" src="portfolioimgs/CSS.png" alt="css"/> <span class="css-li">CSS</span>
+  </li>
+  </ul>
+  <div class="Frameworks">
+    <h4>Frameworks</h4>
+    <img class="scrollDown2" src="portfolioimgs/down2.png" alt="down">
+   </div>
+
+   <div class="Skills">
+    <h4>Skills</h4>
+    <img class="scrollDown2" src="portfolioimgs/down2.png" alt="down">
+   </div>
```

Comment:
- [ ] `Languages`, `Frameworks`, and `Skills` should be list items. As requested in the project requirements. Please fix this :+1: 
![image](https://user-images.githubusercontent.com/28109626/148470991-6f7c7137-dc0a-4ed2-a669-3df03ac31877.png)